packages/core/src/components/essentials/tooltip-axis.ts

```diff
@@ -84,6 +84,15 @@ export class AxisChartsTooltip extends Tooltip {
 					color: this.model.getStrokeColor(datum[groupMapsTo])
 				}
 			];
+
+			// Add radius label and value
+			const { radiusLabel, radiusMapsTo } = Tools.getProperty(options, "bubble");
+			if (radiusLabel) {
```

Comment:
is there anyway for the `bubble graph` component to shoot out a flag to show the radius label rather than tooltip checking for it?

Reply:
also, we'd show the radius item in tooltip where bubble graphs exist regardless of whether `radiusLabel` has been set. the label would be `radiusLabel || "Radius"`

Reply:
> also, we'd show the radius item in tooltip where bubble graphs exist regardless of whether `radiusLabel` has been set. the label would be `radiusLabel || "Radius"`

`radiusLabel` has a default value `Radius`, so the label would be `Radius` when it's not defined in the options.

Reply:
> is there anyway for the `bubble graph` component to shoot out a flag to show the radius label rather than tooltip checking for it?
not sure how ..

Reply:
> is there anyway for the `bubble graph` component to shoot out a flag to show the radius label rather than tooltip checking for it?

not sure how ..

Reply:
> not sure how ..

you'd need to look at how any of the graphs trigger the tooltip to show